llvm/test/SYCLLowerIR/printf_generic_as.ll

```diff
@@ -0,0 +1,102 @@
+;; This tests replacement of string literal address space for __spirv_ocl_printf
+;; at the regular O2 optimization level.
```

Comment:
I suggest to explicitly add `-O2` to the `clang++` command below, in case default optimization level changes in the future